DyldExtractor/Converter/SelectorConvertor.py

```diff
@@ -74,7 +74,8 @@ def convert(self) -> None:
 				selector = self.dyldFile.readString(targetOff)
 
 				if not selector in self.selectorCache:
-					print("\tNo selref for: " + str(selector))
+					if selector != b'\x00':
```

Comment:
We should probably figure out why the selector is b"\x00" but this works for now.